site/docs/v1/tech/apis/identity-providers/_samlv2-request-body.adoc

```diff
@@ -70,9 +70,31 @@ The specified Lambda Id must be of type `SAMLv2Reconcile`.
 
 include::_identity-provider-linking-strategy-request-parameter.adoc[]
 
+[field]#identityProvider.loginHintConfiguration.enabled# [type]#[boolean]# [optional]#Optional#::
+When enabled, at the onset of the AuthN request (using redirect bindings), FusionAuth will attempt to pass a custom specified login hint to the SAML IdP.
+
+[field]#identityProvider.loginHintConfiguration.parameterName# [type]#[String]#::
+When using a login hint, at the onset of the AuthN request (using redirect bindings), the name of the login hint parameter to be provided to the SAML IdP.
+
 [field]#identityProvider.name# [type]#[String]# [required]#Required#::
 The name of this SAML v2 identity provider. This is only used for display purposes. Must be unique across all identity providers.
 
+ifndef::samlv2_idp_initiated[]
+[field]#identityProvider.nameIdFormat# [type]#[String]# [required]#Required# [default]#Defaults to `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent`#::
+The `nameid-format` of this SAML v2 identity provider.
```

Comment:
We try not to re-state the name of the parameter as the description. We should provide more detail than this.